docs/install.rst

```diff
@@ -3,121 +3,120 @@
+    If you plan to import Python 3 projects to your RTD,
+    then you'll need to install Python 3 with virtualenv in your system as well.
 
-    CentOS/RHEL 7 will require::
+In order to get all the dependencies successfully installed,
+you need these libraries.
 
-        sudo yum install python-devel python-pip libxml2-devel libxslt-devel
+On MacOS
+~~~~~~~~
 
-    Users of other Linux distributions may need to install the equivalent
-    packages, depending on their system configuration.
+If you are having trouble on OS X Mavericks
+(or possibly other versions of OS X) with building ``lxml``,
+you probably might need to use Homebrew_ to ``brew install libxml2``,
+and invoke the install with::
 
-.. note::
+    CFLAGS=-I/usr/local/opt/libxml2/include/libxml2 \
+    LDFLAGS=-L/usr/local/opt/libxml2/lib \
+    pip install -r requirements.txt
 
-   If you want full support for searching inside your Read the Docs
-   site you will need to install Elasticsearch_.
+On Ubuntu
+~~~~~~~~~
 
-   Ubuntu users could install this package as following::
+Install::
 
-        sudo apt-get install elasticsearch
+    sudo apt-get install build-essential
+    sudo apt-get install python-dev python-pip python-setuptools
+    sudo apt-get install libxml2-dev libxslt1-dev zlib1g-dev
 
-.. note::
+On CentOS/RHEL 7
+~~~~~~~~~~~~~~~~
 
-   Besides the Python specific dependencies, you will also need Redis_.
+Install::
 
-   Ubuntu users could install this package as following::
+    sudo yum install python-devel python-pip libxml2-devel libxslt-devel
 
-        sudo apt-get install redis-server
+Other Linux
+~~~~~~~~~~~
 
+Users of other Linux distributions may need to install the equivalent packages,
+depending on their system configuration.
 
-You will need to verify that your pip version is higher than 1.5 you can do this as such::
 
-    pip --version
+.. _Python 2.7: http://www.python.org/
+.. _virtualenv: https://virtualenv.pypa.io/en/stable/
+.. _Git: http://git-scm.com/
+.. _`Mercurial`: https://www.mercurial-scm.org/
+.. _Pip: https://pip.pypa.io/en/stable/
+.. _Homebrew: http://brew.sh/
+.. _Elasticsearch: https://www.elastic.co/products/elasticsearch
+.. _Redis: https://redis.io/
 
-If this is not the case please update your pip version before continuing::
 
-    pip install --upgrade pip
+Cloning the repository and start Django
+---------------------------------------
 
-Once you have these, create a virtual environment somewhere on your disk, then
-activate it::
+Clone the repository somewhere on your disk and enter to the repository::
 
-    virtualenv rtd
-    cd rtd
-    source bin/activate
+    git clone https://github.com/rtfd/readthedocs.org.git
+    cd readthedocs.org
 
-Create a folder in here, and clone the repository::
+Create a virtual environment and activate it::
 
-    mkdir checkouts
-    cd checkouts
-    git clone https://github.com/rtfd/readthedocs.org.git
+    virtualenv venv
```

Comment:
Following the _pythonic_ convention here, virtual env called _venv_  and on the project root (now we have this on the `.gitignore` :) #3609).